shims/spark300/pom.xml

```diff
@@ -31,6 +31,46 @@
     <description>The RAPIDS SQL plugin for Apache Spark 3.0.0 Shim</description>
     <version>0.2.0-SNAPSHOT</version>
 
+    <!-- Set 'spark.version' for the shims layer -->
+    <!-- Create a separate file 'SPARK_VER.properties' in the jar to save cudf & spark version info -->
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>dependency</id>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${project.build.directory}/extra-resources"/>
+                                <exec executable="bash" output="${project.build.directory}/extra-resources/${spark300.version}-info.properties">
```

Comment:
it might be good to prefix with spark for the file name spark-300-info.properties

Reply:
Agree, added prefix with `spark` for the version files